tutorials/snap/using-intel-realsense-sdk.md

```diff
@@ -0,0 +1,328 @@
+  - **Body Gestures**: Understands pre-defined body movements. Supports a pointing gesture (including the pointing vector) and a wave gesture.
+  - **Face Tracking**: Tracks face movements. Provides the face direction in values of yaw, pitch, and roll.
+
+
+#### realsense_object_recognition20161208.tar.bz2
+
+Intel® RealSense™ Object Library enables machines to understand what they are viewing and provides meaning to computer vision made possible by Intel RealSense cameras. This ability facilitates more dynamic human machine interaction. Intel® RealSense™ Object Library uses a CNN-based architecture that utilizes depth to efficiently and accurately classify and localize objects. This middleware includes features for recognizing, localizing, and tracking objects from a pre-defined library. 
+
+The following features are available in this release:
+
+  - **Object Recognition (OR)**: Identifies a single object in the scene based on pre-trained classifiers in a pre-defined ROI.
+  - **Object Localization (OD)**: Identifies and locates multiple objects in a scene.
+  - **3D Object Position**: Provides the x,y,z world coordinates of the center of the object. The more exact the bounding box, the more exact this value will be. This is only available in instances where depth is available and will not be provided as a direct output of localization.
+  - **Object Tracking**: Enables the camera to keep track of objects that have been previously localized. 
+
+#### librealsense_slam_20161221.tar.bz2
+
+Intel® RealSense™ SLAM Library provides Simultaneous Localization and Mapping capabilities. 
+
+
+The following features are available in this release: 
+
+  - **Real-time 6 degrees of freedom** (6DoF) camera tracking.
+  - Learning an area by associating its appearance with 6DoF coordinates, so enabling re-localization.
+  - **Real-time construction of a 2D occupancy map** of a captured environment.
+
+SLAM is tailored for, although not limited to, robots:
+Autonomous robot navigation requires the ability to build a high precision map of the surrounding environment and enable the robot to locate itself in the map for path planning and routing purposes.
+
+SLAM uses the visual-inertial sensor of the RealSense camera to estimate odometry and concurrently builds a map. The constructed map can be used by applications to detect obstacles, label locations, and plan paths.
+
+
+#### realsense_sdk_07_12_2016WW50.tar.gz
+
+The Intel® RealSense™ SDK for Linux provides libraries, tools, and samples to develop applications using Intel® RealSense™ cameras, over the Intel librealsense API. 
+
+The SDK provides functionality of record and playback of camera streams for test and validation. 
+
+The SDK includes libraries which support the camera stream projection of streams into a common world-space viewpoint, and libraries which enable the use of multiple middleware modules simultaneously for common multi-modal scenarios.  
+
+It features:
+
+  - **Record and Play**:
```

Comment:
These lists look better in your version than the old version, where the numbering is very confusing.